config/alerting/vmoperator-rules.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMRule
+metadata:
+  name: vmoperator-rules
+spec:
+  groups:
+    - name: vmoperator
+      rules:
+        - alert: LogErrors
+          expr: sum(rate(operator_log_messages_total{level="error", job=~".*((victoria.*)|vm)-?operator"}[5m])) > 0
+          for: 15m
+          labels:
+            severity: high
+            show_at: dashboard
+          annotations:
+            description: "Operator has too many errors at logs: {{ $value}}, check operator logs"
+            dashboard: "{{ $externalURL }}/d/1H179hunk/victoriametrics-operator?ds={{ $labels.dc }}&orgId=1&viewPanel=16"
+            summary: "Too many errors at logs of operator: {{ $value}}"
+        - alert: ReconcileErrors
+          expr: sum(rate(controller_runtime_reconcile_errors_total{job=~".*((victoria.*)|vm)-?operator"}[5m])) > 0
+          for: 10m
+          labels:
+            severity: high
+            show_at: dashboard
+          annotations:
+            description: "Operator cannot parse response from k8s api server, possible bug: {{ $value }}, check operator logs"
+            dashboard: "{{ $externalURL }}/d/1H179hunk/victoriametrics-operator?ds={{ $labels.dc }}&orgId=1&viewPanel=10"
+            summary: "Too many errors at reconcile loop of operator: {{ $value}}"
+        - alert: HighQueueDepth
+          expr: (sum(workqueue_depth{job=~".*((victoria.*)|vm)-?operator"}) by (name)) > 10
```

Comment:
Based on my experience, workqueue_depth for vmuser and vmalertmanager config can be slow and trigger given alert. I propose to exclude it from matching.

 vmuser reconcile is slow because of additional secret creation. The same for alertmanagerconfig

Reply:
done